src/webui/src/components/Overview.tsx

```diff
@@ -58,130 +49,134 @@ class Overview extends React.Component<{}, OverviewState> {
-                                                icon: 'min.png',
-                                                fontColor: titleMinbgcolor
-                                            }}
+                                        <ExpDurationContext.Provider
+                                            value={{ maxExecDuration, execDuration, updateOverviewPage }}
                                         >
-                                            <Title1 />
-                                        </TitleContext.Provider>
-                                    </div>
-                                    <div style={{ position: 'absolute', right: '2%', top: 8 }}>
-                                        <Dropdown
-                                            selectedKey={bestTrialEntries}
-                                            options={entriesOption}
-                                            onChange={this.updateEntries}
-                                            styles={{ root: { width: 170 } }}
-                                        />
+                                            <ExpDuration />
+                                        </ExpDurationContext.Provider>
                                     </div>
-                                </Stack>
-                                <Stack horizontal tokens={stackTokens}>
-                                    <div style={{ width: '40%', position: 'relative' }}>
-                                        <Accuracy
-                                            accuracyData={accuracyGraphData}
-                                            accNodata={noDataMessage}
-                                            height={404}
-                                        />
-                                    </div>
-                                    <div style={{ width: '60%' }}>
-                                        <SuccessTable trialIds={bestTrials.map(trial => trial.info.id)} />
+                                    <div className='empty' />
+                                    <div className='box7'>
+                                        <TitleContext.Provider value={{ text: 'Trial numbers', icon: 'NumberSymbol' }}>
+                                            <Title />
+                                        </TitleContext.Provider>
+                                        <ExpDurationContext.Provider
+                                            value={{ maxExecDuration, execDuration, updateOverviewPage }}
+                                        >
+                                            <TrialCount />
+                                        </ExpDurationContext.Provider>
                                     </div>
-                                </Stack>
-                            </Stack>
+                                </div>
+                                {/* table */}
+                                <div className='box3'>
```

Comment:
why this order of className 'box-'? can they use the same className?

Reply:
They have different style, so can't use same className. Do you mean 'box' is misleading?

Reply:
It will be nice if the class name is more descriptive.

Reply:
fixed.